### ISSUE TITLE
'alltags' Has Missing and Mislabeled Headers on Motus Live

### ISSUE THREAD

**Issue by leberrigan** [MotusDev/Motus-TO-DO#432]
A user has notified me that the code in the Motus R Book wasn't working. Turns out the sqlite database contains the wrong headers on Motus Live, but not Motus Beta.

Table below outlines differences between live and beta versions.

Motus Beta | Motus Live
-- | --
tagDeployStart | _missing_
tagDeployEnd | _missing_
tagDeployLat | tagDepLat
tagDeployLon | tagDepLon
tagDeployAlt | tagDepAlt
tagDeployComments | tagDepComments
recvDeployName | recvDepName
recvSiteName | _missing_

**Comment by Hussell** [comment]
Page reference or link to the code which isn't working?

**Comment by denislepage** [comment]
There may be some un-deployed changes in the R book.

**Comment by leberrigan** [comment]
I can no longer replicate this issue so it may have resolved itself. I have asked researcher who originally notified me of this problem to retry their download and haven't heard back.

The error was associated with an incomplete 'alltags' table that was missing three columns and 5 others were mislabeled (as described above). The code they tried is irrelevant - it only gave them an error because it was referencing those columns that should exist.

User had recently installed R so I thought it have something to do with the installation of motusClient. But I installed the latest version of R and reinstalled all my libraries and still couldn't replicate the problem. Plus, I was experiencing this issue with my older version of R, so I presume this issue was caused by the server from which I was downloading data. Remember, I only manage to get the same error as the user when I switched my dataServerURL back to 'https://motus.org/api/sgdata'.
